media/com_patchtester/js/patchtester.js

```diff
@@ -9,48 +9,34 @@ if (typeof Joomla === 'undefined') {
     throw new Error('PatchTester JavaScript requires the Joomla core JavaScript API')
 }
 
-document.addEventListener("DOMContentLoaded", function (event) {
```

Comment:
type=module or defer means the script executes on `DOMContentLoaded` so this is superfluous